style.css

```diff
@@ -0,0 +1,56 @@
+body{
+    background-image: url(pple.webp);
+}
+#title{
+    margin-left: 450px;
+
+}
+#description{
+    margin-left: 420px;
+    font-size: large;
+    margin-top: 0px;
+    padding-top: 0px;
+}
+form{
+    border: 1px solid rgb(7, 7, 63);
+    background-color: rgba(7, 7, 63, 0.8);
+    border-radius: 10px;
+    color: white;
+    font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+    margin-left: 300px;
+    margin-right: 200px;
+    padding: 40px 0px 40px 50px;
+   
+}
+#Name{
+    width: 600px;
+    padding: 10px;
+   border-radius: 10px;
+  
+}
+#Email{
+    width: 600px;
+    padding: 10px;
+    border-radius: 10px;
+}
+#number{
+    width: 600px;
+    padding: 10px;
+    border-radius: 10px;
+}
+#dropdown{
```

Comment:
Try to give your form element classes. This will make styling your form much easier and it will look presentable